package.json

```diff
@@ -2,7 +2,7 @@
   "name": "xblock-mufi",
   "title": "XBlock MUFI",
   "description": "XBlock for transcribing manuscripts using MUFI font.",
-  "version": "0.1.0",
+  "version": "0.1.8",
```

Comment:
Since we're changing the xblock to use django templating, we can probably make the version `0.2.0`. @stvstnfrd, thoughts?